README.md

```diff
@@ -39,6 +39,16 @@ $ source <virtualenv name>/bin/activate
 $ pip install -r requirements.txt
 ```
```

Comment:
In this section we have discussed about how to reproduce the project. 

Above all these, we should explain how to use it. That is, where to find the nawab twitter and telegram instance and how to get it started

Reply:
We can make that a separate section after the initial version is released?